include/zephyr/device.h

```diff
@@ -884,10 +880,9 @@ static inline bool z_impl_device_is_ready(const struct device *dev)
  */
 #define Z_DEVICE_HANDLES_DEFINE(node_id, dev_id, ...)                          \
 	extern Z_DEVICE_HANDLES_CONST device_handle_t                          \
-		Z_DEVICE_HANDLES_NAME(node_id, dev_id)[];                      \
+		Z_DEVICE_HANDLES_NAME(dev_id)[];                               \
```

Comment:
@JordanYates since you've been recently doing changes in the handles area, can you confirm this makes sense?

Reply:
I believe this is fine. The `elf_parser.py` script links devices and the handles together based on memory addresses, not names. Everything else is done through the handles pointer in the device struct AFAIK.